### PR TITLE
Pokés Gen 7

### PR DESCRIPTION
**Pull request by invalidCards**
Finally got around to adding all of the new friends ~~just in time for SwSh to come around~~. Does not include changed functionality, but does require a version bump because I am too lazy to automate the randomizer. Also some indent fixes ESLint thought necessary.

### CODE REVIEW

**Comment by AprilSylph** [comment]
i don't like the indent changes. was it really eslint complaining? are you sure it was using this repo's eslint rules?

**Comment by invalidCards** [comment]
It was complaining about tabs-as-spaces first, and that is not a rule I enforce myself, so I am fairly sure it was this project's ESLint rules.

**Comment by Wolvan** [comment]
isn't there an eslint fix command that automatically makes the style compliant?

**Comment by invalidCards** [comment]
img

**Comment by AprilSylph** [comment]
what exactly does `npm test` say if you re-indent the html strings?

**Comment by AprilSylph** [comment]
told you it wasn't our eslint rules

**Comment by invalidCards** [comment]
>:(

File: Extensions/pokes.js [hunk 1]
```diff
@@ -419,21 +419,21 @@ XKit.extensions.pokes = {
...
+			'<div class="title">Pokémon</div>' +
+			'<div class="description">Check which Pokémon you have caught</div>' +
 			'</div>' +
 			'<div class="xkit-pokes-menu_button disabled">' +
-				'<div class="icon"></div>' +
-				'<div class="title"></div>' +
-				'<div class="description"></div>' +
+			'<div class="icon"></div>' +
+			'<div class="title"></div>' +
+			'<div class="description"></div>' +
```
Comment (AprilSylph) [inline Extensions/pokes.js]: this html should be indented properly (as it was before)

File: Extensions/pokes.js [hunk 2]
```diff
@@ -136,10 +136,10 @@ XKit.extensions.pokes = {
 
 			if (XKit.extensions.pokes.preferences.allow_fullwidth.value) {
 				poke_html = '<div class="poke ' + poke_class + shiny_class + '" data-pokeid="' + db_nr + '" data-pokesortid="' + poke_sortid + '" data-pokename="' + poke_name + '" data-pokegender="' + poke_gender + '" style="left:' + xpos + 'px; margin-top:' + ypos + 'px;">' +
-				'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
+					'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
```
Comment (AprilSylph) [inline Extensions/pokes.js]: the `'</div>'` should be on a separate line for consistency, or considering the amount of concatenations here, the entire thing would probably be better off as a template string
(also applies to below string)

File: Extensions/pokes.js [hunk 3]
```diff
@@ -135,11 +135,13 @@ XKit.extensions.pokes = {
 			}
 
 			if (XKit.extensions.pokes.preferences.allow_fullwidth.value) {
-				poke_html = '<div class="poke ' + poke_class + shiny_class + '" data-pokeid="' + db_nr + '" data-pokesortid="' + poke_sortid + '" data-pokename="' + poke_name + '" data-pokegender="' + poke_gender + '" style="left:' + xpos + 'px; margin-top:' + ypos + 'px;">' +
-				'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
+				poke_html = `<div class="poke ${poke_class}${shiny_class}" data-pokeid="${db_nr}" data-pokesortid="${poke_sortid}" data-pokename="${poke_name}" data-pokegender="${poke_gender}" style="left:${xpos}px; margin-top:${ypos}px;">` +
+				`<img src="${poke_sprite}" alt="${poke_name}"/>'` +
```
Comment (AprilSylph) [inline Extensions/pokes.js]: `img` element should be indented one, also you forgot to remove the single quote at the end here when converting to template

File: Extensions/pokes.js [hunk 4]
```diff
@@ -135,11 +135,13 @@ XKit.extensions.pokes = {
...
-				'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
+				poke_html = `<div class="poke ${poke_class}${shiny_class}" data-pokeid="${db_nr}" data-pokesortid="${poke_sortid}" data-pokename="${poke_name}" data-pokegender="${poke_gender}" style="left:${xpos}px; margin-top:${ypos}px;">` +
+				`<img src="${poke_sprite}" alt="${poke_name}"/>'` + 
+				'</div>';
 			} else {
-				poke_html = '<div class="poke fixed ' + poke_class + shiny_class + '" data-pokeid="' + db_nr + '" data-pokesortid="' + poke_sortid + '" data-pokename="' + poke_name + '" data-pokegender="' + poke_gender + '" style="margin-top:' + ypos + 'px;">' +
-				'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
+				poke_html = `<div class="poke fixed ${poke_class}${shiny_class}" data-pokeid="${db_nr}" data-pokesortid="${poke_sortid}" data-pokename="${poke_name}" data-pokegender="${poke_gender}" style="margin-top:${ypos}px;">` +
+				`<img src="${poke_sprite}" alt="${poke_name}"/>` +
```
Comment (AprilSylph) [inline Extensions/pokes.js]: indent

**Pull request merged**